android/java/org/chromium/chrome/browser/sync/BraveAndroidSyncSettings.java

```diff
@@ -30,8 +31,11 @@ public void disableChromeSync() { }
 
     // We need to override this to make able
     // DevicePickerBottomSheetContent.createContentView send the link
+    // For Brave we don't have an account in Android system account,
+    // so pretent sync for Brave "account" is alway on when sync is requsted
```

Comment:
typo `pretend`, typo `always`

Reply:
Fixed and rebased https://github.com/brave/brave-core/pull/6417/files#diff-ef378cbdd8f95a5d14d031076e0d68d8R35